staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go

```diff
@@ -666,6 +666,35 @@ func RunTestList(ctx context.Context, t *testing.T, store storage.Interface, com
 		expectRV                   string
 		expectRVFunc               func(string) error
 	}{
+		// Run the non-consistent List first to make sure watch cache hasn't sent any progress notification and it's still on list.ResourceVersion
```

Comment:
Doesn't like that order of test cases matters here, would like to create a separate test. However there is no other way to make sure that this test works with ConsistentListFromCache, as the `t.Parallel()` causes other tests to run and change the state of the feature gate.

Reply:
Filed https://github.com/kubernetes/kubernetes/issues/123677

Reply:
Before we make it run with t.Parallel(), let's not change this test and add a separate test for it. We will merge it once this bug is fixed...